.github/workflows/test_build.yml

```
@@ -0,0 +1,103 @@
+name: OpenDTU-onBattery Test Build
+
+on: workflow_dispatch
+
+jobs:
+  get_default_envs:
+    name: Gather Environments
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
```

actionlint (1.7.12)

[error] the runner of "actions/checkout@v3" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)